test/integration/targets/influxdb_user/tasks/tests.yml

```diff
@@ -81,3 +81,60 @@
       assert:
         that:
           - same_user.changed == false
+
+- name: Test change user password in check mode
+  block:
+    - name: Change user password
+      influxdb_user: user_name=user user_password=user2 login_username=admin login_password=admin
+      check_mode: true
+      register: change_password
+
+    - name: Check that password changing succeeds with a change
+      assert:
+        that:
+          - change_password.changed == true
+
+- name: Test change user password
+  block:
+    - name: Change user password
+      influxdb_user: user_name=user user_password=user2 login_username=admin login_password=admin
+      register: change_password
+
+    - name: Check that password changing succeeds with a change
+      assert:
+        that:
+          - change_password.changed == true
+
+- name: Test remove user in check mode
```

Comment:
I've found that there are no tests to check user removal. So I've added them. And I can move them into separate PR if needed. :-)

Reply:
Integration tests are always welcomed. I think we can keep it here unless there is a specific reason not to.